carbon/app.py

```diff
@@ -34,20 +49,14 @@ def people():
     sql = select([persons.c.MIT_ID, persons.c.KRB_NAME_UPPERCASE,
                   persons.c.FIRST_NAME, persons.c.MIDDLE_NAME,
                   persons.c.LAST_NAME, persons.c.EMAIL_ADDRESS,
-                  persons.c.ORIGINAL_HIRE_DATE, persons.c.HR_ORG_UNIT_TITLE,
+                  persons.c.ORIGINAL_HIRE_DATE, dlcs.c.HR_ORG_LEVEL4_NAME,
                   persons.c.PERSONNEL_SUBAREA_CODE, orcids.c.ORCID]) \
         .select_from(persons.outerjoin(orcids)) \
-        .where(persons.c.EMAIL_ADDRESS != None) \
```

Comment:
To clarify, my understanding is that if a person doesn't have an email address, Elements will skip it. The end result will be the same if we include people without an email address, but I'd rather just get rid of the noise at the start.